crypto/machine_test.go

```diff
@@ -0,0 +1,140 @@
+// Copyright (c) 2020 Tulir Asokan
```

Comment:
You should put your name in things you wrote

Reply:
Sure, changed